lighthouserc.js

```diff
@@ -16,11 +16,7 @@ module.exports = {
   ci: {
     collect: {
       url: getUrls( baseUrl, mobile ),
-      settings: {
-        formFactor: mobile ? 'mobile' : 'desktop',
-        screenEmulation: { mobile: mobile },
-        maxWaitForLoad: 5000
```

Comment:
Are we in danger of running into https://github.com/cfpb/cfgov-lighthouse/pull/30 with this line gone?

Reply:
I thought it was worth giving this simplification a shot, since we're now on a more recent release of Lighthouse.